Gemfile

```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby File.read(".ruby-version").chomp
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 6.0.2"
```

Comment:
I wouldn't drop the pessimistic versioning here - `>=` means you could accidentally bump a major Rails version with `bundle update`

Reply:
Good point.